types/schemas/prototypeApplication/enums/PlanningDesignation.ts

```diff
@@ -174,15 +174,15 @@ type BasePlanningDesignation =
  */
 type NonIntersectingPlanningDesignation = {
   intersects: false;
-} & BasePlanningDesignation;
+} & {value: BasePlanningDesignation};
```

Comment:
`PlanningDesignations` are actually an interesting case where I'm not totally convinced we want to migrate to a new `enum` structure? Will be curious to hear BOPS feedback on this.

Designations & constraints aren't the same typical `value/description` object enums like `ProjectTypes` (which make sense to simplify to string enums!) because _other_ properties are also/still present (eg `intersects`).